python/cudf_polars/cudf_polars/containers/dataframe.py

```diff
@@ -135,7 +137,10 @@ def from_table(cls, table: plc.Table, names: Sequence[str]) -> Self:
             raise ValueError("Mismatching name and table length.")
         return cls(
             # TODO: strict=True when we drop py39
```

Comment:
```suggestion
```

Reply:
grrrr thought I caught all these, thank you